### ISSUE TITLE
About th evalaution on DPDD 

### ISSUE THREAD

**Issue by notorious-eric** [codeslake/IFAN#1]
Hi, I notice that in your paper, the test results of DPDNet is different from the original paper, where in the original paper PSNR is 25.13,and in your paper PSNR is 25.23. I wonder what causes the difference?

**Comment by codeslake** [comment]
Hi!

PSNR in the main paper of DPDNet is measured with the DPDNet trained and tested with 8bit images, but all the models we've compared in our paper are trained and tested with 8-bit images.

The PSNRs that we have reported in our paper are measured from the checkpoint of DPDNet(_dual) trained with 8-bit images that we acquired from the authors. But it is still different from what they have reported in their supplementary material (Table 5). We just reported higher PSNR in our paper.

Also, since the authors of DPDNet did not have the checkpoint of DPDNet(_single) trained with 8-bit images, we've trained the network with the training code provided by the authors.

Thanks
